### ISSUE TITLE
Midpoint Review [DGIB, 1990 Form Updates for MGIB]

### ISSUE THREAD

**Issue by rlyons89** [department-of-veterans-affairs/va.gov-team#34780]
# Steps to complete Midpoint Review 

## Before meeting

**VFS Product Manager**
- [X]  Create this issue and schedule meeting via Calendly (you are responsible for adding the VFS attendees). 
- [X] Notify the Collaboration Cycle team of this ticket with scheduled date/time in #vfs-platform-support ([see Slack Workflow User guide](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html)). 
- [ ]  Add artifacts listed below to the ticket at least 2 days before scheduled meeting. 

**Platform**
- [ ] Add the Platform reviewers to scheduled meeting
- [ ] Add Zoom information to scheduled meeting (if synchronous)
- [ ] Create feedback tickets and link to the Midpoint Review epic (if no feedback, please comment stating that). 

## After meeting
**VFS Product Manager**
- [ ] Respond to feedback tickets.
- [ ] Close this ticket when review is complete and feedback has been addressed.  

**Platform**
- [ ] Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=266151061). 

### Has your team met with the Sitewide Content team for content support? 
- [ ] Yes*
- [X] No

\*If yes, please include any relevant content artifacts.

## Artifacts

Provide links or documents for the following: 

- [X] Product outline : https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/my-education-benefits/MGIB%20Updates/DGIB%20Product%20Outline%20-%20MGIB%20Updates.docx
 
- [ ] Finalized design prototype or mockup
- [ ] Research plan 

 
- [ ] Conversation guide 

### Content artifacts 

- [ ] Content brief (if applicable) and your product’s content source of truth 

- [name of content of truth, for example, "Invision Prototypes"](link here)

- [ ] Github ticket for any relevant static content page and entry point updates for tool or feature (if applicable)
 
### QA artifacts

[Learn more about QA artifacts](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/qa-artifacts.md#test-plan)

- [ ] Regression test plans (if there isn't already a regression test plan for this feature in TestRaiProduct)
- [ ] Link to test cases/test plan in TestRail (draft OK) 

## Asynchronous review option
Would you like this review to be asynchronous? 

Please refer to the [Midpoint Review guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html) for the difference between a synchronous meeting and an asynchronous review.
- [ ] Yes
- [X] No

## Scheduling
- You will need to use Calendly to schedule both synchronous and asynchronous reviews (asynchronous reviews will not have a Zoom link).
- Open the [Calendly midpoint review calendar](https://calendly.com/collaboration-cycle/design-intent-or-midpoint-review)
- Select a date and time and click “Confirm”
- Add your name and email
- Click "Add Guests" and enter the VFS meeting attendees email addresses. 
- Click Schedule Event

## Additional information
Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Midpoint Review Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html) on Platform website for more information about the Collaboration Cycle.

**Comment by chaseakirby** [comment]
Review has been scheduled for January 14th, 2022 at 2:00 PM EST. Please forward to any additional VFS attendees if needed.

**Comment by jimuhh** [comment]
Hi @rlyons89 Do you have any prototypes and/or research plans for us to review at this touchpoint? If so, please link to them in this document. Thanks.

**Comment by rlyons89** [comment]
@jimuhh yes, sorry didn't see this message earlier, here is a board with our latest designs and the flow between the screens : https://accenturefederal.invisionapp.com/freehand/Chapter-30-33-1606-Application-Flow-pmgSp7sDh

**Comment by shiragoodman** [comment]
the Midpoint Review has been postponed to Tuesday 1/18 at 11:00am ET

**Comment by joshkimux** [comment]
# Accessibility Feedback 
@rlyons89 and @shiragoodman apologies for the late feedback on a holiday as I just returned from a 2 week break-- I'm posting all possible feedback below knowing that many of these points may be out of context or scope having missed prior conversations. Do let me know what does (or does not) apply and I'm happy to edit this accordingly ✌️ 

## Must
- For effective date "Chapter 30", duplicate "learn more" buttons within the radios will likely cause accessibility defects and will require additional testing/engineering work. If possible, I would recommend to explore simpler patterns. More than happy to talk about what that might look like 🙇 

## Should
- For the bolded "you cannot change your decision...", given that it is critical information, it may be worth planning for / redesigning the experience to account for screen reader users specifically. For example, visually placing it as hint text below the label (and semantically _within_ the `<label>`).
- For "Chapter 1606"... "Montgomery GI Bill..." is styled as a heading for visual rather than semantic effect (in this instance). It may be worth exploring alternative ways of designing this to visually and semantically communicate importance. For example, using a single card with an alert icon and alt text.
- For contact information page 1/3, "go to our profile" link should include purpose. For example, writing the link within the note as "Note: you can `update your contact information [purpose] on your profile [target]` for other VA benefits."
- "Download your certificate of eligibility..." should be styled as a download link, not a button. If the link below ( "Certificate of Eligibility") has the same behavior, it should have the same link text that includes purpose [download your] and target [certificate of eligibility].
- Review links on the conclusion page for purpose and target. For example, `figure out your career path with CareerScope` is more descriptive as a link compared to `CareerScope`. For screen reader users who navigate solely using links (through the link list), having purpose and target included is valuable.  

## Consider
- [Reviewing inclusive research strategies](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/research/recruitment.md#), specifically how might we include underserved communities into our qualitative measures (for example, people with disabilities / who use assistive technology)?
- The headings on the authenticated application instructions page may be redundant. Specifically the `h3` "Start your application and the `h2` begin your application for education benefits. I'll defer to the IA/content team here!
- Instead of asking users to confirm their email address in two separate text fields (which can cause barriers to folks with dyslexia or other cognitive disabilities), it may be easier to ask for it once with a email holding a confirmation link.

## Design to development recommendations
- For "step 2 of 5, choose the benefit..." page, use `aria-describedby` to link the chapter titles to their respective headings for screen reader users who may jump to different sections using a headings list.
- Ensure alt text is included for icons that convey meaning. For example, an `sr-only` text (e.g. `<span class="sr-only">Alert</span>` can be used to provide meaningful text to the warning (!) icon next to "You are not eligible for this benefit."
- For effective date "Chapter 30", the alert card with "if you give up..." appears to be dynamically populated post selection of a radio option. In this case, a polite `aria-live` will be necessary to announce that it has been populated to screen reader users. A less complicated alternative could be to populate an expandable learn more below since it would have a natural tab stop.

## Questions
- Does this midpoint review include the how to apply page? If so:
  - Would it be possible to explore alternative patterns to the dynamic "find your education benefits form" wizard? For context-- [this pattern is being deprecated by the design system team](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/399#issuecomment-990011476) for accessibility and usability reasons.
  - If the "Go to GI Bill Website" is a link, it should be styled as a link.
  - The links under "Other VA benefits..." should be underlined so folks with colorblindness (specifically on mobile) can still understand they are links. 
- For "step 2 of 5, choose the benefit..." page, will Veterans understand what the tags within the cards (e.g. tuition and fees, money for housing, etc) mean/refer to?
- For Contact information 3/3, what if a Veteran does not have a mobile number? Would the notification section not populate?
- For the truncated application, is "restart your VA benefits application..." a button (doing something) or a link (going somewhere)? If it's a button, it should be styled as a button.
